srcpkgs/nvimpager/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'nvimpager'
+pkgname=nvimpager
+version=0.6
+revision=1
+wrksrc="${pkgname}-${version}"
+build_style=gnu-makefile
+make_build_args="PREFIX=/usr"
+make_build_target="nvimpager.configured"
+make_install_args="PREFIX=/usr"
+hostmakedepends="neovim curl git pandoc"
```

Comment:
It seems they are fetching thing using curl in https://github.com/lucc/nvimpager/blob/develop/makefile#L36, which isn't great. If you could download this in the template as a versioned distfile, which allows us to checksum it, that would be much better.

Reply:
I'm not totally sure I understand how they use git, but you should probably try to check the output package, the version info could be looking weird.

Reply:
Curl is used to get `AnsiEsc.vba` which I added as a distfile. I moved it to `build_wrksrc` in `pre_build()`. Is there a better way to do this?

Git is used get the date of the last commit for `metadata.yml`. I ended up hardcoding this in `pre_build()` aswell.

Reply:
The way you did it is great, thanks! Are you sure the file in that link doesn't change? The build could fail in the future due to the checksum changing. Btw, the links need to be in different lines, like the checksums.

Reply:
Perhaps leave a comment about the commit date needing to be changed in the case of any update?

Reply:
I switched to the release version of `AnsiEsc.vba` from https://www.vim.org/scripts/script.php?script_id=302